project.clj

```diff
@@ -21,6 +21,7 @@
                  [com.stuartsierra/component "0.3.1"]
                  [com.taoensso/timbre "4.7.0"]
                  [digest "1.4.4"]
+                 [kixi/kixi.comms "0.1.6" :exclusions [cheshire]]
```

Comment:
where is the cheshire conflict? Feels a bit odd excluding something from our own libs.